src/gluonnlp/models/transformer.py

```diff
@@ -441,26 +441,26 @@ def __init__(self, units: int = 512,
                                                      num_heads=num_heads,
                                                      attention_dropout=self._attention_dropout,
                                                      dtype=dtype,
-                                                     layout=attention_layout)
-        self.proj_in = nn.Dense(units=units, in_units=units, flatten=False,  use_bias=False,
+                                                     layout='NTK')
+        self.proj_in = nn.Dense(units=units, in_units=units, flatten=False,  use_bias=True,
                                 weight_initializer=weight_initializer,
                                 bias_initializer=bias_initializer,
                                 dtype=dtype)
         self.attn_inter_q = nn.Dense(units,
                                      in_units=units,
-                                     use_bias=False,
+                                     use_bias=True,
```

Comment:
Does BART always use True? I think the other models use False.

Reply:
In decoder, bart use bias for the contextual attention but not for self attention

Reply:
I see. We should add some flags to make sure the existing codes won't break.

Reply:
Sorry, bart uses bias for both contextual attention and self attention in decoder. Could we add a flag `use_qkv_bias` to set this up?

Reply:
Yes, we should add this flag.